config/ufs/config_grids.xml

```diff
@@ -116,6 +151,19 @@
   </required_gridmaps>
 
   <gridmaps driver="nuopc">
+    <gridmap atm_grid="T62" ocn_grid="Atlantic8">
```

Comment:
Are you adding mapping files because there is a bug in ESMF? And we are planning to remove these? If so can you please add a comment and point to the ESMF issue? If not - then these mapping files should be removed.

Reply:
Actually, i am adding those mapping files but I am not using them to test the mode. I was thinking that it is a convention to add them to the XML file. So, if it now required, I could remove them. BTW, we have still hanging issue when we are generating routhandle for patch interpolation and there is a active ESMF ticket about it. The current implementation uses bilinear to overcome the issue temporarily.

Reply:
The goal is to have no mapping files hard-wired unless they are required (like the smoothing map from rof->ocn). So please remove these hardwired mapping.

Reply:
I try to remove them but i am getting error like following in the XML validation. Do I need to set them "unset"?

```
ERROR: Command: '/usr/bin/xmllint --xinclude --noout --schema /glade/u/home/turuncu/UFS/ufs-hafs-app/cime/config/xml_schemas/config_grids_v2.1.xsd /glade/u/home/turuncu/UFS/ufs-hafs-app/cime/config/ufs/config_grids.xml' failed with error '/glade/u/home/turuncu/UFS/ufs-hafs-app/cime/config/ufs/config_grids.xml:3: element grid_data: Schemas validity error : Element 'grid_data': Missing child element(s). Expected is ( gridmaps ).
/glade/u/home/turuncu/UFS/ufs-hafs-app/cime/config/ufs/config_grids.xml fails to validate' from dir '/glade/u/home/turuncu/UFS/ufs-hafs-app/cime/scripts'
```

Reply:
you should not need to add anything new in <required_gridmaps> other than what was there originally.

Reply:
Actually, I did not touch <required_gridmaps> part. Those are coming from UFS release. I also checked the CESM one (master branch) and <required_gridmaps> section also exists there.